maquetado/scss/detail.scss

```diff
@@ -106,10 +127,21 @@ $body-padding-top-for-book: 200px - $body-padding-top-for-back;
   padding-top: $body-padding-top-for-book;
 }
 
+.rectangle {
+  height: 2px;
+}
+
 .separated-right {
   margin-right: 40px;
 }
 
+.sign-up-button {
```

Comment:
the login and signup button should have the same height and border radius, maybe you can use a mixin: https://sass-lang.com/documentation/at-rules/mixin